src/components/TrackTile/TrackerDetails/to-observation-resource.ts

```diff
@@ -44,7 +44,7 @@ const toFhirObservationResource = (
     meta: {
       tag: [
         {
-          code: accountSettings?.project || '',
```

Comment:
This addresses feedback about a possible 400 when submitting an observation with an empty string.